docs/pages/using-nhs-notify/text-message-templates.md

```diff
@@ -0,0 +1,100 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: page
+title: Text message templates
+parent: Using NHS Notify
+nav_order: 3
+permalink: /using-nhs-notify/text-message-templates
+section: Writing a message
+---
+
+Upload a Word document of your text message and NHS Notify will send it for you.
+
+## How to edit your text message using our template
+
+Download our [text message template](/assets/worddocs/Text%20message%20template%20NHS%20Notify%202.docx) to set up your email.
+
+You can save the template and edit it with your message content.
+
+## Formatting your text message
+
+Organise your message content using the styles in the template.
+
+If you’re copying and pasting text from another document, paste it without any formatting. Then [apply the styles](https://support.microsoft.com/en-gb/office/apply-styles-f8b96097-4d25-4fac-8200-6139c8093109) in the template.
+
+You can also use [Microsoft Word’s format painter](https://support.microsoft.com/en-gb/office/use-the-format-painter-4bb415a9-d4e4-42b7-b579-170adc594e40) to copy the formatting of the styles and apply it to your text.
+
+Find out how to use:
+
+- [personalisation](#personalisation)<!-- markdownlint-disable-line -->
+- [headings](#headings)
+- [body text](#body-text)
+- [links](#links)
+- [bullet points](#bullet-points)
```

Comment:
Remove this line as do not have section on bullet points

Reply:
Removed unnecessary bullets and headings anchor links.